package.json

```diff
@@ -27,5 +27,6 @@
         "url": "https://github.com/Microsoft/tslib.git"
     },
     "main": "tslib.js",
+    "jsnext:main": "tslib.es6.js",
```

Comment:
how is this supposed to work on node versions that does not support `const`? i do not think rollup rewrites `const` to `var`?